pkg/whiteflag/confirmation.go

```diff
@@ -132,13 +132,15 @@ func ConfirmMilestone(cachedMsBundle *tangle.CachedBundle, forEachConfirmedTx fu
 	// confirm all txs of the included tails
 	for _, txHash := range mutations.TailsIncluded {
 		if err := forEachBundleTxWithTailTxHash(txHash, func(tx *tangle.CachedTransaction) {
-			tx.GetMetadata().SetConfirmed(true, milestoneIndex)
-			tx.GetMetadata().SetRootSnapshotIndexes(milestoneIndex, milestoneIndex, milestoneIndex)
-			conf.TxsConfirmed++
-			conf.TxsValue++
-			metrics.SharedServerMetrics.ValueTransactions.Inc()
-			metrics.SharedServerMetrics.ConfirmedTransactions.Inc()
-			forEachConfirmedTx(tx, milestoneIndex, confirmationTime)
+			if !tx.GetMetadata().IsConfirmed() {
```

Comment:
why would this be true?

Reply:
Is it due to the funny bundle tx reuse from discord?

Reply:
Because of f*cking bundles.... and their tail reattachments, you would count all the single txs of this bundle several times in the confirmation.